### PR TITLE
fixing css from majors components to  assembling them

### PR DESCRIPTION
**Pull request by BBriset**
css fix

### CODE REVIEW

**Review by BenZen59 — DISMISSED** [review]

File: src/App.jsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ function App() {
     <div className="main">
       <BrowserRouter>
         <Switch>
-          <Route path="/" exact>
```
Comment (LoicBrassart) [inline src/App.jsx:15]: I'm "almost" sure this is an error, and was not tested (plus, it has nothing to do withthe PR's title)

File: src/components/searchBar/SearchBar.jsx [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 import './Search.css';
 
-const Search = () => {
+const SearchBar = () => {
   const [query, setQuery] = useState('');
   const [results, setResults] = useState([]);
 
   const onChange = (e) => {
     e.preventDefault();
+
```
Comment (LoicBrassart) [inline src/components/searchBar/SearchBar.jsx]: Useless empty line
+ the previous line is useless too: a "change" event has no default behaviour to cancel :shrug:

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Please fix the blocking error

**Review by BenZen59 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
No more Evil

**Pull request merged**